components/constants/contracts/MyBitToken.js

```diff
@@ -1,5 +1,5 @@
 module.exports = {
-  ADDRESS: '0x391aaD2823B9008A44FB54E018c97603A8fcb2c8',
+  ADDRESS: '0x5d60d8d7eF6d37E16EBABc324de3bE57f135e0BC',
```

Comment:
So probably it's worth having a way to pick networks and contracts.

Reply:
Yea it is, that is the goal but due to time constraints I say we stick to mainnet for now. And also the setting up of our own provider, that still needs to be tested.